README.md

```diff
@@ -1,163 +1,163 @@
-## 简介
-PaddleOCR旨在打造一套丰富、领先、且实用的OCR工具库，助力使用者训练出更好的模型，并应用落地。
-
-**近期更新**
-- 2020.6.8 添加[数据集](./doc/datasets.md)，并保持持续更新
-- 2020.6.5 支持 `attetnion` 模型导出 `inference_model`
-- 2020.6.5 支持单独预测识别时，输出结果得分
-- 2020.5.30 提供超轻量级中文OCR在线体验
-- 2020.5.30 模型预测、训练支持Windows系统
+## Introduction
```

Comment:
It is recommended to put the English readme in the README_EN.md file, to distinguish it from the Chinese README